src/styles/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  box-sizing: border-box;
+  width: calc(#{$boxSize} * 7 + #{$gapSize} * 6 + 20px);
```

Comment:
Mathematical calculations can also be transferred to a variable and use the finished result.